sirepo/template/srw.py

```diff
@@ -1893,16 +1898,29 @@ def _rotate_report(report, ar2d, x_range, y_range, info):
     return ar2d, x_range, y_range
 
 
+def _rsopt_res_file_infix(elements, params):
+    s = ''
+    for e in elements:
+        for p in [p for p in params if p in e]:
```

Comment:
This is a lot of concepts in one line. I think this would be easier to read:

```py
for p in params:
    if p in e:
        for i, x in enumerate(e[p].fieldNames):
```

Reply:
fb40f8e